json_to_df.py

```diff
@@ -0,0 +1,116 @@
+
+import json
+import os
+
+
+import pandas as pd
+
+
+from unidecode import unidecode
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+# 
+# Loop through raw UGB JSON files and pull out texts for analysis in R with
+# tex2vec package.
+# 
+
+DIR = 'C:/Users/ben_ryan/Documents/DARPA UGB'
+bfiles = [file for file in os.listdir(DIR) if ('json' in file and 'breitbart' in file)]
+bcomments = []
+barts = []
+
+#
+# Process all comments into one DataFrame
+# 
+
+for file in bfiles:
+    file = open('{}/{}'.format(DIR, file), 'r')
+    data = json.load(file)
+
+    for date in data.keys():
+        for i, article in enumerate(data[date]):
+            title = article['title']
+            author = article['author']
+            art_id = '.'.join([date,str(i)])
+
+            bcomments = bcomments + [(date, 
+                                      title,
+                                      author,
+                                      art_id,
+                                      comment['commenter'], 
+                                      comment['parent_commenter'], 
+                                      comment['comment'], 
+                                      comment['upvotes']) for comment in article['parsed_comments']]
+
+bc = pd.DataFrame(bcomments)
+bc.columns = ['date',
+              'art_title',
+              'art_author',
+              'art_id',
+              'commenter',
+              'parent',
+              'comment_txt',
+              'upvotes']
+
+#
+# Process all articles into one DataFrame
+# 
+
+for file in bfiles:
+    file = open('{}/{}'.format(DIR, file), 'r')
+    data = json.load(file)
+
+    for date in data.keys():
+            barts = barts + [(date, 
```

Comment:
This seems to work fine, but you might be able to get most of the parsing done using the `pd.DataFrame.from_dict()` call with a little less fuss. However, if it works and there's no real degradation in performance, no big deal. Something to keep in mind for the future.